deploy/charts/bottlerocket-shadow/templates/custom-resource-definition.yaml

```diff
@@ -0,0 +1,187 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    cert-manager.io/inject-ca-from: brupop-bottlerocket-aws/root-certificate
+  name: bottlerocketshadows.brupop.bottlerocket.aws
+spec:
+  conversion:
+    strategy: Webhook
+    webhook:
+      clientConfig:
+        service:
+          name: brupop-apiserver
+          namespace: brupop-bottlerocket-aws
+          path: /crdconvert
+          port: 443
```

Comment:
This needs to be templated with the values defined in the `values.yaml` for this chart 👀

Reply:
Done! ☑️